packages/react-adminer/src/components/SectionBox.tsx

```diff
@@ -134,7 +134,7 @@ const SectionBox: React.FC<Props> = ({
 				}
 
 				return (
-					<WithCol key={f.name}>
+					<WithCol key={f.name} sm={f.grid}>
```

Comment:
to se mi moc nezdá, opravdu to funguje? čekal bych že v tom `WithCol` to nějak použiješ, ale nevidím to.

Reply:
Funguje
![image](https://user-images.githubusercontent.com/109352229/197505762-cde04617-00d5-448f-a9ba-e7312a74c828.png)

![image](https://user-images.githubusercontent.com/109352229/197505842-74c419c8-b852-46be-9493-dfa69b88b798.png)

Reply:
serialNumber má grid 24
color má grid 8
ostatní nemaj nastavenej grid
A vypadá to, že to funguje dobře.

Reply:
aha, tohohle sem se nevisml, super